Patches.py

```diff
@@ -2943,6 +2943,4 @@ def patch_rupee_tower(location, rom: Rom):
 def patch_ganons_tower_bk_door(rom: Rom, flag):
     var = (0x05 << 6) + (flag & 0x3F)
     bytes = [(var & 0xFF00) >> 8, var & 0xFF]
-    rom.write_bytes(0x2EE30FE, bytes)
-
-
+    rom.write_bytes(0x2EE30FE, bytes)
```

Comment:
You deleted one newline too many. By convention, all text files end with exactly one newline.

Reply:
addressed